cranelift/codegen/src/machinst/lower.rs

```diff
@@ -1201,6 +1201,20 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
             _ => false,
         }
     }
+
+    pub fn block_successor_label(&self, block: Block, succ: usize) -> MachLabel {
+        trace!("block_successor_label: block {block} succ {succ}");
+        let lowered = self
+            .vcode
+            .block_order()
+            .lowered_index_for_block(block)
+            .expect("Unreachable block");
+        trace!(" -> lowered block {lowered:?}");
+        let (_, succs) = self.vcode.block_order().succ_indices(lowered);
+        trace!(" -> succs {succs:?}");
+        let succ_block = *succs.get(succ).expect("Successor index out of range");
+        MachLabel::from_block(succ_block)
+    }
```

Comment:
In `verify_try_call_handler_index` below there's an offset-by-1 to skip what I presume is the default normal-return path, but this doesn't seem to also have a -1 or adjust-by-1 anywhere. Is that intentional?